salalib/importutils.cpp

```diff
@@ -174,7 +174,11 @@ namespace depthmapX {
             QtRegion region;
 
             for(auto& point: points) {
-                region = runion(region, point);
```

Comment:
Hmm, on second thought, is that necessary? What does runion do on a null pointer?

Reply:
See below. The odd thing is that "isNull()" doesn't check for nullness, only if all the coordinates (bottom left x, y and top right x, y) are 0.0. I'll rename it so that it makes more sense.
```
QtRegion runion(const QtRegion& a, const QtRegion& b)
{
   QtRegion n;
   n.bottom_left.x = a.bottom_left.x < b.bottom_left.x ?
                     a.bottom_left.x : b.bottom_left.x;
   n.bottom_left.y = a.bottom_left.y < b.bottom_left.y ?
                     a.bottom_left.y : b.bottom_left.y;
   n.top_right.x   = a.top_right.x   > b.top_right.x ?
                     a.top_right.x   : b.top_right.x;
   n.top_right.y   = a.top_right.y   > b.top_right.y ?
                     a.top_right.y   : b.top_right.y;
   return n;
}
```